client/core/trade_simnet_test.go

```diff
@@ -758,9 +758,10 @@ func TestResendPendingRequests(t *testing.T) {
 		for !foundSwapErrorNote {
 			select {
 			case note := <-notes:
-				foundSwapErrorNote = note.Severity() == db.ErrorLevel && note.Subject() == SubjectSwapError
+				foundSwapErrorNote = note.Severity() == db.ErrorLevel && (note.Subject() == SubjectSwapSendError ||
+					note.Subject() == SubjectInitError || note.Subject() == SubjectReportRedeemError)
 			case <-time.After(4 * time.Second):
-				return fmt.Errorf("client %d: no init/redeem error note after 4 seconds", client.id)
+				return fmt.Errorf("client %d: no init/redeem error note after 1 second", client.id)
```

Comment:
It does have a 4 second timeout

Reply:
Whoops. Rebase conflict resolution mistake.